lxd/lifecycle/lifecycle.go

```diff
@@ -0,0 +1,17 @@
+package lifecycle
```

Comment:
You should also add it to the list of packages in the static analysis test suite (test/suites/static_analysis.sh).